pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/CollectiveDataEdition/adapters/getVenueCollectiveDataAdapter.ts

```diff
@@ -0,0 +1,29 @@
+import { GET_DATA_ERROR_MESSAGE } from 'core/shared'
+import { GetCollectiveVenueResponseModel } from 'apiClient/v1'
+import { api } from 'apiClient/api'
+
+type GetVenueCollectiveDataAdapter = Adapter<
+  string,
+  GetCollectiveVenueResponseModel,
+  null
+>
+
+const getVenueCollectiveDataAdapter: GetVenueCollectiveDataAdapter =
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏On unit test pas les adapteurs ?